contribute/style-guides/backend.md

```diff
@@ -75,3 +75,15 @@ Valid reasons to use a pointer include (but not necessarily limited to):
   allocating heap memory)
 * You might *need* `nil` to tell if a variable isn't set, although usually it's better to use the type's zero
   value to tell instead
+
+## Database
+
+### Foreign keys
+
+While they can be useful, we don't generally use foreign key constraints in Grafana, for historical and
+technical reasons. See this [comment](https://github.com/grafana/grafana/issues/3269#issuecomment-383328548) by Torkel 
+for context.
```

Comment:
```suggestion
While they can be useful, we don't generally use foreign key constraints in Grafana, for historical and
technical reasons. Refer to this [comment](https://github.com/grafana/grafana/issues/3269#issuecomment-383328548) for context.
```

Reply:
@achatterjee-grafana All your suggestion does here is join two lines. Am I breaking a style guideline wrt. Markdown line length? Lines shorter than a certain limit are more practical to read, especially when diffing.

Reply:
It is an internal document, a style guide. If the break helps you read, we can go with that.

Reply:
I don't really understand. I was wondering if you suggested the change, because a style guideline dictates a longer line length.

Reply:
I'm not sure if we have a standard for Markdown line lengths, but I think they should generally be kept shorter than a certain limit (~100-120 chars).

Reply:
there is one guideline seems:
`Basic conventions for Markdown files
Wrap all lines at 80 characters. Denote bold text using the asterisk format: **bold text**`

Reply:
@ying-jeanne Do you have a link for that?

Reply:
We do not anything specific in the [markdown style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-markdown-guide.mdl).

Reply:
If it's a _standard_ Markdown convention you're referring to, we're probably not following it. 80 sounds a bit conservative anyhow.

Reply:
@achatterjee-grafana In that case, we can resolve this.

Reply:
I doubt there is no offical guideline for it, seems a lot of projects put 80-100 characters https://www.google.com/search?q=official+markdown+line+length&rlz=1C1CHBF_enFR925FR925&sxsrf=ALeKk03doieh4V3RkA-KwYGpl138driGIg:1610464412361&ei=nLz9X_LSFcOaa7S2qFg&start=0&sa=N&ved=2ahUKEwjykOO515buAhVDzRoKHTQbCgs4ChDy0wN6BAgJEDU&biw=1920&bih=937